src/CodeAnalysis.ruleset

```diff
@@ -72,6 +72,149 @@
     <Rule Id="CA5371" Action="None" /> <!-- Use XmlReader For Schema Read -->
     <Rule Id="CA5372" Action="None" /> <!-- Use XmlReader For XPathDocument -->
   </Rules>
+  <Rules AnalyzerId="StyleCop.Analyzers" RuleNamespace="StyleCop.Analyzers">
+    <Rule Id="AD0001" Action="None" /> <!-- Analyzer threw an exception -->
+    <Rule Id="SA0001" Action="None" /> <!-- XML comments -->
```

Comment:
📝 If this is disabled, all SA16xx rules are allowed to freely misbehave. You'll want to make sure those stay off.

Reply:
> You'll want to make sure those stay off.

They will.